### PR TITLE
Remove compiler warnings

### PR DESCRIPTION
**Pull request by raymondng**
@andrewconner @stkem

### CODE REVIEW

File: kifi-backend/modules/eliza/app/com/keepit/eliza/MessageRepo.scala [hunk 1]
```diff
@@ -57,33 +59,18 @@ class MessagesForThread(val thread:Id[MessageThread], val messages:Seq[Message])
...
-      }
-    }
-    def writes(mft: MessagesForThread):JsValue = {
-      Json.obj(
-          "thread_id"  -> mft.thread.id,
-          "messages"   -> JsArray(mft.messages.map{Json.toJson(_)})
-        )
-    }
-  }
+object MessagesForThread {
```
Comment (andrewconner) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/MessageRepo.scala:62]: :+1:

File: kifi-backend/modules/eliza/app/com/keepit/eliza/MessageRepo.scala [hunk 2]
```diff
@@ -48,7 +49,8 @@ object Message {
     )(Message.apply, unlift(Message.unapply))
 }
 
-class MessagesForThread(val thread:Id[MessageThread], val messages:Seq[Message]) {
+case class MessagesForThread(val thread:Id[MessageThread], val messages:Seq[Message])
+{
```
Comment (stkem) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/MessageRepo.scala:53]: Ahhhh, open brace on a new line!!?!?! ;-)

**Comment by stkem** [comment]
lgtm

File: kifi-backend/modules/eliza/app/com/keepit/eliza/MessageRepo.scala (hunk 2 above)
Comment (yingjieMiao) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/MessageRepo.scala:53]: my first textbook in C++ back in 2002 did that style. So nostalgic.

**Comment by eishay** [comment]
thanks!

**Pull request merged**